cognite_toolkit/_cdf_tk/commands/clean.py

```diff
@@ -309,13 +306,12 @@ def execute(
         for loader_cls in reversed(resolved_list):
             if not issubclass(loader_cls, ResourceLoader):
                 continue
-            loader = loader_cls.create_loader(ToolGlobals)
+            loader = loader_cls.create_loader(ToolGlobals, build_dir)
             if type(loader) is DataSetsLoader:
                 self.warn(ToolkitNotSupportedWarning(feature="Dataset clean."))
                 continue
             result = self.clean_resources(
                 loader,
-                build_path / loader_cls.folder_name,
```

Comment:
This is an example for a simplified call to `clean_resources` as the loadrs already knows the build path it is not necessary any more